### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by swaraj7**

### CODE REVIEW

**Comment by vishesh92** [comment]
There are some conflicts. Please resolve them. And there are multiple commits with the same message.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,64 @@
...
+
+`POST /snippet`
+
+**Input**
+
+```
+{
+    "snippet_name" : "unique name for the snippet",
+    "snippet_privacy" : true, 
+    "snippet_expiry" : 5D,
```
Comment (vishesh92) [inline README.md:18]: I guess `5D` should be a string.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,64 @@
...
+**Input**
+``` 
+{
+    "snippet_id" = 1234
+}
+```
+
+**Output**
+```
+Status : 302 Found
```
Comment (vishesh92) [inline README.md:54]: The status code series 3xx is for Redirection types. In this case the status code should be 200.

File: file_sender.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import json
```
Comment (vishesh92) [inline file_sender.py:1]: Is this file required?

File: mysite/mysite/settings.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
+"""
```
Comment (vishesh92) [inline mysite/mysite/settings.py:1]: Please change the name for the site.

File: mysite/ShareBox/views.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    return HttpResponse("this is an index page")
+
+
+def send_data(request, snippet_data):
+    not_empty = bool(snippet_data)
+    if not_empty is False:
+        raise "BucketEmptyError"
+    else
+        requests.post("http://www.shareboxes.herokuapp.com/share", data=snippet_data)
+        retrun HttpResponse("Data Uploaded")
```
Comment (vishesh92) [inline mysite/ShareBox/views.py]: typo

File: mysite/mysite/settings.py [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+# Application definition
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'ShareBox',
```
Comment (dash1291) [inline mysite/mysite/settings.py:40]: module names should be lowercase
